workspace.go

```diff
@@ -380,6 +386,15 @@ type WorkspaceCreateOptions struct {
 	// exist, it is created and added to the workspace.
 	Tags []*Tag `jsonapi:"relation,tags,omitempty"`
 
+	// Struct containing boolean values that indicate whether the resolved value of
+	// a setting should be decided by something other than the workspace (i.e. a related
+	// organization or project).
+	//
+	// For example: When setting the execution-mode of a workspace to "default", the
+	// `setting-overwrites.execution-mode` field should be set to false, because the execution-mode
+	// will be overwritten by either the organization or project.
```

Comment:
There are a few problems with this doc string: 

- There's no such value as "default", but it makes it look like there might be. 
- The "whether" is backwards! When describing a bool, the "whether" condition should align with the `true` value and the "or not" condition with the `false`. 

How about:

```suggestion
	// Optional: Struct of booleans, which indicate whether the workspace
	// specifies its own values for various settings. If you mark a setting as
	// `false` in this struct, it will clear the workspace's existing value for
	// that setting and defer to the default value that its project or
	// organization provides.
	//
	// In general, it's not necessary to mark a setting as `true` in this
	// struct; if you provide a literal value for a setting, Terraform Cloud will
	// automatically update its overwrites field to `true`. If you do choose to
	// manually mark a setting as overwritten, you must provide a value for that
	// setting at the same time. 
```

Reply:
Lovely!